utils/src/test/java/com/cloud/utils/net/MacAddressTest.java

```diff
@@ -51,10 +51,4 @@ public final void testMacAddressToLong() throws Exception {
     // TODO    public final void testToString() throws Exception {
     // TODO    public final void testGetMacAddress() throws Exception {
     // TODO    public final void testParse() throws Exception {
```

Comment:
Thanks for removing those TODOs. Just curious about the others above.
Why not remove all TODOs?

Reply:
Got it. Thanks, @DaanHoogland.